storybook/stories/contributing/contributing-get-ready.txt

```diff
@@ -0,0 +1,87 @@
+
+Then, you’ll need to connect your GitHub and WordPress.org accounts to make sure your contributions are correctly attributed. It’ll take just two minutes if you follow these steps:
+
+[Connect GitHub and WP.org accounts](https://make.wordpress.org/core/handbook/tutorials/linking-your-github-and-w-org-profiles/)
+
+#### **Step 3: Learn about the contribution process** {#step-3:-learn-about-the-contribution-process}
+
+In order to make effective contributions to WPDS, it’s important to know the basics of our contribution process. While we’ll take you through a step-by-step process depending on your contribution, we’ll explain the basics below.  
+   
+Depending on the contribution you’re making to the design system, you’ll either need to create an issue, or both an issue and a pull request (PR). 
+
+**What’s an issue?**
+
+Issues are how we keep track of changes to our design system on GitHub. You can find [all open issues on the Gutenberg repository](https://github.com/WordPress/gutenberg/issues).
+
+Smaller changes, like fixing a typo, don’t require you to create a PR yourself, and the implementation can be handled by the community. In this case, you can just create a new issue and add a clear description of the work you’re doing.
+
+Make sure to add the Design System label when creating an issue, and consider adding one of the following labels when needed:
+
+Needs Design — Needs design efforts  
+Needs Copy Review — Needs review of user-facing copy (language, phrasing)  
+Needs Dev — Ready for, and needs developer efforts  
+Needs Accessibility Feedback — Needs input from accessibility
+
+Reviewers will chime in to discuss if your change is worth adding to the design system. When it’s accepted, someone will gladly help you with pushing the change.
+
+**What’s a pull request?**
+
+Pull requests are how you propose changes to the design system on GitHub. You can find [all open PRs on the Gutenberg repository](https://github.com/WordPress/gutenberg/pulls).
+
+If you’re making a bigger change to the design system, you’re expected to make the change to the design system yourself. Think of documenting a new pattern, adding a component, or overhauling an existing component.
+
+First, you create an issue to track all work related to your change, with a clear description and the right tags. Reviewers will chime in to decide if your change is worth pursuing, and once it’s been greenlit, you can start working on it.
+
+When you’re ready to share what you’ve been working on, you can add a PR to your issue to show your proposed changes. Creating a 
+
+#### **Step 4: Create your first PR** {#step-4:-create-your-first-pr}
+
+Creating a PR takes a couple of steps that involve copying the Gutenberg repository to your device.
+
+You can find a step-by-step explanation [on this page](https://developer.wordpress.org/block-editor/contributors/code/git-workflow/#git-workflow-walkthrough). If you’re running into any issues, let us know in #design-systems on Slack.
```

Comment:
> let us know in #design-systems on Slack.

Same as above. I think we need to settle on a public channel for feedback when we can.